scripts/actions/mobskills/heat_barrier.lua

```diff
@@ -13,9 +13,9 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    -- TODO: Enfire power, Blaze Spikes reduced power in Salvage zones
-    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BLAZE_SPIKES, math.random(50, 67), 0, 180))
-    -- xi.mobskills.mobBuffMove(mob, xi.effect.ENFIRE, ???, 0, 180)
+    -- TODO: Blaze Spikes reduced power in Salvage zones
+    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BLAZE_SPIKES, mob:getMainLvl() * 0.8, 0, 180))
+    xi.mobskills.mobBuffMove(mob, xi.effect.ENFIRE, mob:getMainLvl() * 0.4, 0, 300)
```

Comment:
Shouldnt this operations be floored? I dont think we want floats in power

Reply:
I thought that might have been the case as well but we actually read it as a double and then cast it to uint16 which essentially floors it. So this is technically okay but I'm happy to floor it here as well.

https://github.com/LandSandBoat/server/blob/base/src/map/lua/lua_baseentity.cpp#L12781

Reply:
Nah, its ok. If it gets floored down the line, I'm happy